core/src/main/java/org/elasticsearch/search/aggregations/bucket/range/geodistance/GeoDistanceParser.java

```diff
@@ -164,6 +162,11 @@ public AggregatorFactory parse(String aggregationName, XContentParser parser, Se
         return new GeoDistanceFactory(aggregationName, vsParser.config(), InternalGeoDistance.FACTORY, origin, unit, distanceType, ranges, keyed);
     }
 
+    private static String getSearchParseExceptionMessage(String currentFieldName, String aggregationName) {
+        return "Failed when parsing field: '" +  currentFieldName +
```

Comment:
People cannot infer anything from the message "Unexpected token START_OBJECT" .
Here token is START_OBJECT .... which does not convey anything to the user and more that that it is just "misleading" . thats why removed it ......
why should we have some thing that will confuse the user ...
I think it is only for internal usage for  parser (equivalent to the symbols in the json content) ...
so removed it .. 
reply ok ?